### PR TITLE
Update simulation

### PR DESCRIPTION
**Pull request by Yadunund**
* updated visuals from latest traffic editor
* updated workcell description

### CODE REVIEW

File: src/simulation.md [hunk 1]
```diff
@@ -307,27 +306,34 @@ Analogous to the `door_supervisor`, a `lift_supervisor` [node](https://github.co
...
+Robots performing deliveries within facilities is a common use case and hence a `Delivery` task is configured into the `rmf_fleet_adapters`. In a delivery task, a payload is loaded onto the robot at one location and unloaded at another. The loading and unloading of the payload may be automated by a robot/workcell in the facility. To integrate these systems with RMF core systems, a set of dispenser messages are [defined](https://github.com/osrf/rmf_core/tree/master/rmf_dispenser_msgs/msg). Despite their names, the messages are generalized for use by systems other than dispensers as well. When a robot reaches the loading station, its `rmf_fleet_adapter` publishes a `DispenserRequest` message which the station receives and begins processing. When the loading is successful, the station publishes a `DispenserResult` message with `SUCCESS` status. The `rmf_fleet_adapter` then instructs the robot to proceed to the unloading station where a similar set of messages are exchanged with the workcell unloading the payload.
```
Comment (mdfesto) [inline src/simulation.md]:
```suggestion
Robots performing deliveries within facilities is a common use case and hence a `Delivery` task is configured into the `rmf_fleet_adapters`. In a delivery task, a payload is loaded onto the robot at one location and unloaded at another. The loading and unloading of the payload may be automated by a robot/workcell in the facility. To integrate these systems with RMF core systems, a set of dispenser messages are [defined](https://github.com/osrf/rmf_core/tree/master/rmf_dispenser_msgs/msg). Despite their names, the messages are generalized for use by systems other than dispensers as well. When a robot reaches the loading station, its `rmf_fleet_adapter` publishes a `DispenserRequest` message which the station receives and begins processing. When the loading is successful, the station publishes a `DispenserResult` message with `SUCCESS` status. The `rmf_fleet_adapter` then instructs the robot to proceed to the unloading station where a similar set of messages are exchanged with the workcell unloading the payload.
```

Comment (mdfesto) [reply src/simulation.md]: Are `station` and `waypoint` interchangeable here? Or is one a location and one is a function? I'm getting a little mixed up on terminology. @Yadunund for your advice

File: src/simulation.md [hunk 2]
```diff
@@ -307,27 +306,34 @@ Analogous to the `door_supervisor`, a `lift_supervisor` [node](https://github.co
...
 
-To replicate this delivery behavior in simulation, the `TeleportDispenser` and `TeleportIngestor` [plugins](https://github.com/osrf/rmf_demos/tree/master/rmf_gazebo_plugins/src) are designed. These plugins are loaded into [3D models](https://github.com/osrf/rmf_demos/tree/master/rmf_demo_assets/models) the same names.
-To setup a payload loading station:
-* Assign a `workcell_name` attribute to the waypoint (see figure below)
-* Add a `TeleportDispenser` model beside the waypoint with `name` matching the `workcell_name`
+To replicate the loading and unloading processes in simulation, the `TeleportDispenser` and `TeleportIngestor` [plugins](https://github.com/osrf/rmf_demos/tree/master/rmf_gazebo_plugins/src) are designed. These plugins are attached to the `TeleportDispenser` and `TeleportIngestor` [3D models](https://github.com/osrf/rmf_demos/tree/master/rmf_demo_assets/models) respectively.
```
Comment (mdfesto) [inline src/simulation.md]:
```suggestion
To replicate the loading and unloading processes in simulation, the `TeleportDispenser` and `TeleportIngestor` [plugins](https://github.com/osrf/rmf_demos/tree/master/rmf_gazebo_plugins/src) have been designed. These plugins are attached to the `TeleportDispenser` and `TeleportIngestor` [3D models](https://github.com/osrf/rmf_demos/tree/master/rmf_demo_assets/models), respectively.
```

File: src/simulation.md [hunk 3]
```diff
@@ -307,27 +306,34 @@ Analogous to the `door_supervisor`, a `lift_supervisor` [node](https://github.co
...
 
-To setup a payload unloading station:
-* Assign a `workcell_name` attribute to the waypoint (see figure below)
-* Add a `TeleportIngestor` model beside the waypoint with `name` matching the `workcell_name`
+To setup a payload unloading station in simulation:
+* Add a `TeleportIngestor` model beside the dropoff waypoint and assign it a
+  unique `name`.
 
-When a `DispenserRequest` message is published with `target_guid` matching the name of the `TeleportDispenser` model, the plugin will teleport the payload onto the nearest robot model. Conversely, when the `target_guid` matches the name of the `TeleportIngestor` model, the `TeleportIngestor` plugin will teleport the payload on the robot to its location in the world. The combinations of these plugins allow for delivery requests to be simulation. In the future, this mechanism will be replaced by actual workcells or robot arms but the underlying message exchanges will remain the same.
+When a `DispenserRequest` message is published with `target_guid` matching the
```
Comment (mdfesto) [inline src/simulation.md:321]: I see the term repeated but is `target_guid` intentionally leaving off the trailing `e`?

**Review by mdfesto — COMMENTED** [review]
Looks great, a few comments/edits for your consideration

File: src/simulation.md (hunk 3 above)
Comment (Yadunund) [reply src/simulation.md:321]: `target_guid` is a field in the DispenserRequest message definition
https://github.com/osrf/rmf_core/blob/66b34c5968392f0e9515d69edc9489570fbeae23/rmf_dispenser_msgs/msg/DispenserRequest.msg#L7

File: src/simulation.md (hunk 1 above)
Comment (Yadunund) [reply src/simulation.md]: Sorry I totally mixed up my terminology here. There are pickup and dropoff waypoints with loading and unloading workcell at each respectively. I have removed instances of `station` and updated the para https://github.com/osrf/ros2multirobotbook/pull/25/commits/1b04be24cd34e4eefc5c74855877204f855f8acd

**Review by mdfesto — APPROVED** [review]

File: src/simulation.md (hunk 3 above)
Comment (mdfesto) [reply src/simulation.md:321]: thx!

**Pull request merged**